src/hooks/settingsHooks/useGetSettings.ts

```diff
@@ -127,17 +127,26 @@ const extractPassword = (
 }
 
 export const useGetSettings = (
-  siteName: string
+  siteName: string,
+  isEmailLogin: boolean
 ): UseQueryResult<SiteSettings> => {
   return useQuery<SiteSettings>(
     [SETTINGS_CONTENT_KEY, siteName],
     async () => {
       const siteSettings = await SettingsService.get({ siteName })
-      // TODO: reimplement actual functionality once netlify issue resolved
-      // const passwordSettings = await SettingsService.getPassword({ siteName })
-      const passwordSettings = {
-        isAmplifySite: false,
-        password: "",
+      let passwordSettings
+      if (isEmailLogin) {
+        // TODO: reimplement actual functionality once netlify issue resolved
+        // passwordSettings = await SettingsService.getPassword({ siteName })
+        passwordSettings = {
+          isAmplifySite: false,
+          password: "",
+        }
+      } else {
+        passwordSettings = {
+          isAmplifySite: false,
+          password: "",
+        }
```

Comment:
Hey, in both control flows the password settings get set to the same value, is this intended?

[Style, nit]: Since we have version control, I prefer not to have commented code in production

Reply:
As discussed offline - we're making this clearer that the branch is blocked off until launchDarkly! https://github.com/isomerpages/isomercms-frontend/pull/1344/commits/83e780e3743cf8b573cf942a0ba576ddf03f316d